components/site/patterns/Create Patterns/CreateServiceModal.js

```diff
@@ -0,0 +1,122 @@
+import * as React from 'react';
```

Comment:
I may have missed this from another PR, but can the folder name not include spaces? `create-patterns` ideally